test/jdk/java/awt/Frame/WindowMoveTest.java

```diff
@@ -53,7 +54,9 @@ public static void main(String[] args) throws Exception {
         EventQueue.invokeAndWait(() ->
                 frame.dispatchEvent(new WindowEvent(frame, WindowEvent.WINDOW_CLOSING)));
 
-        WindowMove.latch.await();
+        if (!WindowMove.latch.await(10, TimeUnit.SECONDS)) {
```

Comment:
I think 10 seconds is too long… We usually use 2 seconds for various delays.

Reply:
I agree, If it doesn't happen in much less than 10 seconds probably it'll never happen.